src/Riok.Mapperly/Configuration/MapperConfiguration.cs

```diff
@@ -9,23 +9,25 @@ namespace Riok.Mapperly.Configuration;
 /// Newly added properties must also be added to <see cref="Riok.Mapperly.Helpers.MapperConfigurationBuilder"/>.
```

Comment:
Rename reference of `MapperConfigurationBuilder`

Reply:
It should be `MapperConfigurationMerger`

Reply:
Fixed2 😊